ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh

```diff
@@ -36,12 +36,19 @@ cp /tmp/ssh-privatekey ~/.ssh/id_rsa
 chmod 0400 ~/.ssh/id_rsa*
 
 # Set up the pull secret in the expected location
-export PULL_SECRET="\${HOME}/.pull-secret.json"
-cp /tmp/pull-secret "\${PULL_SECRET}"
+export PULL_SECRET="${HOME}/.pull-secret.json"
+cp /tmp/pull-secret "${PULL_SECRET}"
 
 cd ~/microshift
 
 ./test/bin/ci_phase_iso_build.sh
+
+mkdir /tmp/osbuilder-logs
+workers_services=$(sudo systemctl list-units | awk '/osbuild-worker@/ {print $1} /osbuild-composer\.service/ {print $1}')
+while IFS= read -r service; do
+    journalctl -u "${service}" &> /tmp/osbuilder-logs/${service}.log
+done <<<"${workers_services}"
```

Comment:
Can we add this to the ci_phase_iso_build in case it fails so that it drops it to the same dir where all output goes? That way it gets automatically picked up by the trap in line 68.

Reply:
Sounds good. Will close and open the PR on openshift/microshift.